src/client/App.js

```diff
@@ -57,7 +58,10 @@ function App() {
         <Route exact path="/order-confirmation">
           <OrderConfirmationPage />
         </Route>
-
+        {/* Testing Paypal page path,should be removed  */}
+        <Route exact path="/paypal">
+          <PaypalPage />
+        </Route>
```

Comment:
Paypal path should go to authenticate route!

Reply:
I remove this path, its only for testing the component.